Frameworks/Foundation/NSDecimalNumber.mm

```diff
@@ -312,34 +350,98 @@ - (NSDecimalNumber*)decimalNumberByRaisingToPower:(NSUInteger)power {
  @Status Stub
  @Notes
 */
-- (NSDecimalNumber*)decimalNumberByMultiplyingByPowerOf10:(short)power {
+- (instancetype)initWithString:(NSString*)numericString locale:(id)locale {
     UNIMPLEMENTED();
     return StubReturn();
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
+*/
+- (NSDecimal)decimalValue {
+    // Read-Only
+    NSDecimal decimal = { 0 };
```

Comment:
> 0  [](start = 25, length = 3)

? IN general prefer the use of empty {} initializers as {0} will only 0 init the first field in a struct I think.